src/main/scala/Recursion1.scala

```diff
@@ -0,0 +1,10 @@
+object Recursion1 extends App {
+    var count = 0
+
+    def recur(word:String,times:Int){
```

Comment:
var count = 0 shouldn't be a variable and instead be in the method like this

def recur(word:String,times:Int, count: Int){

and then as you'll be calling it just increment it by one